tecken/tests/test_upload.py

```diff
@@ -1081,15 +1105,21 @@ def adjusted_getmtime(fn):
 
         monkeypatch.setattr(os.path, "getmtime", adjusted_getmtime)
 
-        # Run the command
-        call_command("remove_orphaned_files", verbose=True)
+        with sentry_helper.reuse() as sentry_client:
+            # Run the command
+            call_command("remove_orphaned_files", verbose=True)
 
-        # Assert message is logged
-        assert f"error getting size: {str(path)}" in caplog.text
-        assert "FileNotFound" in caplog.text
+            # Assert message is logged
+            assert f"error getting size: {str(path)}" in caplog.text
+            assert "FileNotFound" in caplog.text
 
-        # Assert metric is emitted
-        incr_records = metricsmock.filter_records(
-            "incr", stat="tecken.remove_orphaned_files.delete_file_error"
-        )
-        assert len(incr_records) == 1
+            # Assert a Sentry event was emitted
+            event = sentry_client.events[0]
+            assert event["logentry"]["message"] == "error getting size: %s"
+            assert event["logger"] == "tecken.remove_orphaned_files"
+
+            # Assert metric is emitted
+            incr_records = metricsmock.filter_records(
+                "incr", stat="tecken.remove_orphaned_files.delete_file_error"
```

Comment:
Ah yeah, this is more what I'm used to seeing in tests. I mean sometimes we do make assertions about what's logged, but those are typically secondary to what functions are/aren't being called, whether an error is thrown, etc.